### ISSUE TITLE
ng add @ngrx/store@latest --minimal false does not create a reducers.ts file

### ISSUE THREAD

**Issue by shaun-jacks** [ngrx/platform#2998]
## Minimal reproduction of the bug/regression with instructions:

Create a new angular application with the following commands:
` npx @angular/cli@11.2.9 new --routing --strict --style scss`

Within root directory of new angular project, call ng-add schematics for @ngrx/store with minimal set to false:

`yarn ng add @ngrx/store --minimal false`

Using yarn as a package manager
```
yarn --version
1.21.1
```

## Expected behavior:

The schematics should create a `reducers/index.ts` file in the same folder as `app.module.ts`.

Right now, it is importing 

`import { reducers, metaReducers } from './reducers';`

In `app.module.ts` without having a defined `reducers/index.ts` file, so this fails to compile.

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):

Package json snippet
```json
"dependencies": {
    "@angular/animations": "~11.2.10",
    "@angular/common": "~11.2.10",
    "@angular/compiler": "~11.2.10",
    "@angular/core": "~11.2.10",
    "@angular/forms": "~11.2.10",
    "@angular/platform-browser": "~11.2.10",
    "@angular/platform-browser-dynamic": "~11.2.10",
    "@angular/router": "~11.2.10",
    "@ngrx/store": "11.1.1",
    "rxjs": "~6.6.0",
    "tslib": "^2.0.0",
    "zone.js": "~0.11.3"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "~0.1102.9",
    "@angular/cli": "~11.2.9",
    "@angular/compiler-cli": "~11.2.10",
    "@types/jasmine": "~3.6.0",
    "@types/node": "^12.11.1",
    "codelyzer": "^6.0.0",
    "jasmine-core": "~3.6.0",
    "jasmine-spec-reporter": "~5.0.0",
    "karma": "~6.1.0",
    "karma-chrome-launcher": "~3.1.0",
    "karma-coverage": "~2.0.3",
    "karma-jasmine": "~4.0.0",
    "karma-jasmine-html-reporter": "^1.5.0",
    "protractor": "~7.0.0",
    "ts-node": "~8.3.0",
    "tslint": "~6.1.0",
    "typescript": "~4.1.5"
  }
```

Node version: `v12.16.1`

## Other information:

A workaround that seems to work for now is using `yarn ng generate @ngrx/schematics:store State --root --module app.module.ts` which does similar functionality, except also imports `StoreDevtoolsModule`

I have noticed that this command succeeds for 
`@ngrx/store: version "10.1.2"`

## I would be willing to submit a PR to fix this issue

[ ] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

**Comment by shaun-jacks** [comment]
I am also just noticing the same behavior for

```
yarn ng add @ngrx/effects --minimal false
```

Where it does not create an `app.effects.ts` file

**Comment by timdeschryver** [comment]
This is by design, the `minimal` flag sets up a module but doesn't create reducers/effects/...
See the [docs](https://ngrx.io/guide/schematics/store#options) for more info.

**Comment by timdeschryver** [comment]
My bad, I just noticed you wrote `--minimal false`.
Feel free to open a PR with a fix @shaun-jacks .

**Comment by shaun-jacks** [comment]
@timdeschryver just created a PR that should fix this #3003 !
